anchor/network/src/discovery.rs

```diff
@@ -21,23 +23,63 @@ use libp2p::swarm::{
 };
 use lighthouse_network::discovery::enr_ext::{QUIC6_ENR_KEY, QUIC_ENR_KEY};
 use lighthouse_network::discovery::DiscoveredPeers;
-use lighthouse_network::{CombinedKeyExt, Subnet};
+use lighthouse_network::CombinedKeyExt;
 use tokio::sync::mpsc;
 use tracing::{debug, error, warn};
 
-use lighthouse_network::EnrExt;
-
 use crate::Config;
-
+use lighthouse_network::EnrExt;
+use serde::{Deserialize, Serialize};
+use ssz::{Decode, Encode};
+use ssz_types::length::Fixed;
+use ssz_types::typenum::U128;
+use ssz_types::{BitVector, Bitfield};
+
+/// Target number of peers to search for given a grouped subnet query.
+const TARGET_PEERS_FOR_GROUPED_QUERY: usize = 6;
```

Comment:
We may be able to simplify some of this. I don't think the SSV subnets are dynamic, like if we have X,Y,Z operators, do we need to constantly search for subnets? Perhaps we can just subscribe to them forever?

Reply:
How often we subscribe will depend on how frequently new committees are created for a peer. However, after subscribing, the subnet membership includes all peers assigned to that subnet number (CommitteeID % 128), not just peers from the same committee.